### DISCUSSION TITLE
All SEARCH RESULTS SENT in log contain "-bryan adams -housezzz"

### DISCUSSION THREAD

**Discussion by kennethb22** [slskd/slskd#1096] [Q&A] [answered]
See log excerpt below. What's going on here? Thanks!

**Comment by kennethb22** [comment]
16:06:35	INF	["SEARCH RESULT SENT"]: Sending 572 records to "cailleaumichel" for query '"lounge -bryan adams -housezzz"'
16:06:34	INF	["SEARCH RESULT SENT"]: Sending 9 records to "Insuadibilis" for query '"stayin alive -bryan adams -housezzz"'
16:06:33	INF	["SEARCH RESULT SENT"]: Sending 2 records to "SebKy" for query '"one love flac -bryan adams -housezzz"'
16:06:33	INF	["SEARCH RESULT SENT"]: Sending 4 records to "mwelter" for query '"gaskin -bryan adams -housezzz"'
16:06:33	INF	["SEARCH RESULT SENT"]: Sending 123 records to "Laddie86" for query '"genesis music -bryan adams -housezzz"'
16:06:32	INF	["SEARCH RESULT SENT"]: Sending 2 records to "SkaarjMaster" for query '"black sabbath back street kids -bryan adams -housezzz"'
16:06:31	INF	["SEARCH RESULT SENT"]: Sending 219 records to "Malevolo" for query '"judas priest -bryan adams -housezzz"'
16:06:31	INF	["SEARCH RESULT SENT"]: Sending 201 records to "cristian1972" for query '"voyage voyage -bryan adams -housezzz"'
16:06:31	INF	["SEARCH RESULT SENT"]: Sending 14 records to "magola" for query '"Norah Jones -bryan adams -housezzz"'

**Comment by jpdillingham** [comment, marked as answer]
This is an unintended (and unanticipated!) side effect of #1089.  The phrases "bryan adams" and "housezzz" are search phrases sent from the server for which clients are not supposed to send results.  To comply with this I add these phrases as exclusions to incoming searches, and what you're seeing is the result of that.

I have an issue somewhere to stop logging searches this way, and I'll need to make sure that's done soon to stop this from happening.

Thanks for bringing it to my attention!

**Reply by kennethb22** [reply]
Thank you!

**Comment by jpdillingham** [comment]
#1097 adjusts the logic so these additional terms won't be logged any longer.
